packages/config/config/devices/0x015f/mh-dt311_411.json

```diff
@@ -449,19 +459,33 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Disabled",
+					"label": "Disable",
 					"value": 0
 				},
 				{
-					"label": "Enabled",
+					"label": "Enable",
 					"value": 1
 				}
 			]
 		},
+		"35": {
```

Comment:
We're making changes to preserve the order of the file in the UI. To me it makes sense for the delay to come after the enable/disable.

Reply:
agreed